payroll-processor-client/src/app/employee/employee-list/state/employee-list.model.ts

```diff
@@ -0,0 +1,39 @@
+export interface Employee {
```

Comment:
We can model the different types in the client like we have in the functions now - `Employee` and `EmployeeNew`.

`EmployeeNew` wouldn't have an `id` or `version` field.

We'll want to add `version` to the `Employee` type.

Reply:
done!